cli-feed-v4.json

```diff
@@ -41,12 +41,12 @@
       "hidden": false
     },
     "v4": {
-      "release": "4.30.0",
+      "release": "4.31.0",
```

Comment:
Given the templates are for a now GA product, is it okay to update both `v4` and `v4-prerelease`?

Reply:
We have set up the process to let the changes bake in for a few days. VS code has some nightly tests that run on pre-release and prod tags. We have been dinged for moving directly to prod and then running into issues.

Reply:
Okay, so update prerelease only?